projects/netgrif-components-core/src/lib/toolbar/abstract-toolbar.component.ts

```diff
@@ -30,4 +39,32 @@ export abstract class AbstractToolbarComponent {
     activeLang(lang: string): boolean {
         return this.translate.currentLang === lang;
     }
+
+    public logout(): void {
+        this.userService.logout().subscribe(() => {
+            if (!!this.toolbarConfig.loginUrl && this.toolbarConfig.loginUrl !== "") {
+                this.router.navigate([this.toolbarConfig.loginUrl]);
+            } else {
+                this.router.navigate([this.redirectService.resolveLoginPath()]);
+            }
+        });
+    }
+
+    public profile(): void {
+        this.router.navigate([this.toolbarConfig.profileUrl]);
```

Comment:
this shouldnt be from static config, but from nae.json, add property to nae.json (like the login redirect is, it is required property), if the config isnt available, profile button shouldnt be visible, this property shouldnt be from process ,because process dont configure profile management and the user doesnt need to know on which url is profile configured, it should be at nae.json because its responsible for routing and component management